.ci/on_conan_release.jenkinsfile

```diff
@@ -0,0 +1,87 @@
+/*
+
+    Triggered whenever a new Conan version is uploaded to pypi.
+
+*/
+
+node('Linux') {
+    stage('Input parameters') {
+        echo """
+        - conan_version: ${params.conan_version}
+        - build_new_images: ${params.build_new_images}
+        - build_old_images: ${params.build_old_images}
+        """
+    }
+
+    stage('New docker images') {
+        // Trigger the build (no need to wait)
+        if (params.build_new_images) {
+            build(job: 'ConanCenterIndex', wait: false)
```

Comment:
Don't you have to pass the conan_version to the job or the latest version is picked automatically?

Reply:
I believe it will be filled by the Conan Release Jenkins job.

Reply:
The `ConanCenterIndex` one takes the latest dockerfiles from the repository and generates images for the latest Conan versions from pypi. That way we ensure that all of them are identical (except for the Conan version).

So here there is no need for that version.